src/Nethermind/Nethermind.Runner/Ethereum/Steps/EthereumStepsManager.cs

```diff
@@ -180,10 +194,22 @@ private async Task RunOneRoundOfInitialization()
                 }
 
                 Stopwatch stopwatch = Stopwatch.StartNew();
-                await step.Execute();
-                _hasFinishedExecution[stepBaseType] = true;
-                stopwatch.Stop();
-                if (_logger.IsInfo) _logger.Info($"Step {step.GetType().Name.PadRight(24)} executed in {stopwatch.ElapsedMilliseconds}ms");
+                Task task = step.Execute().ContinueWith(t =>
+                {
+                    _hasFinishedExecution[stepBaseType] = true;
```

Comment:
What about exceptions?

Reply:
this part is not finished - EthereumSubsystems should handle exceptions